### ISSUE TITLE
With buttons moved to tree, tree frame bottom shadow has not full height

### ISSUE THREAD

**Issue by Deesen** [pmfx/ElementsInTree#9]
You need to add `parent.tree.resizeTree();` after `jQuery("#treeMenu").detach().prependTo("#tabDoc");`

**Comment by pmfx** [comment]
Thanks.
